jplag/src/main/java/de/jplag/reporting/reportobject/model/ComparisonReport.java

```diff
@@ -35,27 +20,33 @@ public ComparisonReport(String firstSubmissionId, String secondSubmissionId, flo
         this.matches = matches;
     }
 
-    public String getFirstSubmissionId() {
+    @Override
+    public String firstSubmissionId() {
         return firstSubmissionId;
     }
 
-    public String getSecondSubmissionId() {
+    @Override
+    public String secondSubmissionId() {
         return secondSubmissionId;
     }
 
-    public float getMatchPercentage() {
+    @Override
+    public float matchPercentage() {
         return matchPercentage;
     }
 
-    public List<FilesOfSubmission> getFilesOfFirstSubmission() {
+    @Override
+    public List<FilesOfSubmission> filesOfFirstSubmission() {
         return filesOfFirstSubmission;
     }
 
-    public List<FilesOfSubmission> getFilesOfSecondSubmission() {
+    @Override
+    public List<FilesOfSubmission> filesOfSecondSubmission() {
         return filesOfSecondSubmission;
     }
 
-    public List<Match> getMatches() {
+    @Override
+    public List<Match> matches() {
         return matches;
     }
```

Comment:
I am not the biggest expert on Java records but don't they generate the accessors automatically? Why do we have to explicitly specify them?

Reply:
Yes .. I'll have a look at it :D

Reply:
Resolved